source/maintainer-guide/installing/installing-from-docker.rst

```diff
@@ -0,0 +1,47 @@
+.. _installing-from-docker:
+
+Installing from docker
+######################
+
+
+Overview
+--------
+
+Docker is a popular virtualisation technology in the web hosting domain. 
+Docker can also be used to try out new software without the need to install various dependencies.
+The GeoNetwork community maintains 'Docker images' ready to run in your Docker environment. 
+These images are available at 'Docker hub': https://hub.docker.xom/_/geonetwork.
+Docker desktop includes a tool called Kitematic, which can be used to download and run docker images from Docker hub.
+This article guides you in getting started with GeoNetwork by using Kitematic and Docker.
+
+Before you Start
+----------------
+
+Install the latest version of Docker desktop including Kitematic from https://www.docker.com/products/docker-desktop.
+
+Steps to get started with GeoNetwork
+------------------------------------
+
+Start the kitematic application. On some operating systems Kitematic will now start a 'Docker VM', in which the docker images will run.
+
+Use the search bar to search in docker hub with a keyword 'GeoNetwork'. Click the 'create' button on the official GeoNetwork card.
+
+.. figure:: img/kitematic-search.png
+
+Kitematic will download the docker image and start it in the docker environment. This will take a couple of minutes, depending on your internet bandwith.
```

Comment:
be careful of providing commentary (documenting what is happening), it is not a separate step.

```
#. Use the :guilabel:`search bar` to search in docker hub with a keyword :kbd:`GeoNetwork`.

#. Click the : guilabel:`create` button on the official GeoNetwork card.
   
   Docker download the image, and start it as a container in your docker environment. This will take a couple of minutes, depending on your internet bandwith.
```